test/extended/operators/olm.go

```diff
@@ -230,4 +233,215 @@ var _ = g.Describe("[sig-operator] an end user use OLM", func() {
 		}
 
 	})
+
+	// OCP-21126 - Subscription status says 'CSV is installed' when it is not
+	// author: scolange@redhat.com
+	g.It("Subscription status says CSV is installed", func() {
```

Comment:
In the g.It("") we're supposed to have the OCP & the priority
ex: 	`g.It("**OLM-Medium-OCP-21126-**Subscription status says CSV is installed", func() {`

The **-** locations matter too.